localization/ndt_scan_matcher/include/ndt_scan_matcher/ndt_scan_matcher_core.hpp

```diff
@@ -195,6 +200,9 @@ class NDTScanMatcher : public rclcpp::Node
   std::shared_ptr<Tf2ListenerModule> tf2_listener_module_;
   std::unique_ptr<MapModule> map_module_;
   std::unique_ptr<PoseInitializationModule> pose_init_module_;
+
+  bool estimate_scores_for_degrounded_scan_;
+  double z_margin_for_ground_removal_ = 1.0;
```

Comment:
```suggestion
  double z_margin_for_ground_removal_;
```
I think it is better to avoid providing default value here